dotnetv3/StepFunctions/Tests/StepFunctionsWrapperTests.cs

```diff
@@ -0,0 +1,152 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+using Amazon.StepFunctions.Model;
+
+namespace SupportTests
+{
+    public class StepFunctionsWrapperTests
+    {
+        private readonly IConfiguration _configuration;
+        private readonly string _activityName;
+        private readonly string _roleArn;
+        private static string _activityArn;
+        private static string _executionArn;
+        private static string _stateMachineArn;
+        private readonly string _stateMachineName;
+
+        private static IAmazonStepFunctions _client;
+        private static StepFunctionsWrapper _wrapper;
+
+        /// <summary>
+        /// Constructor for the test class.
+        /// </summary>
+        public StepFunctionsWrapperTests()
+        {
+            _configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("testsettings.json") // Load test settings from .json file.
+                .AddJsonFile("testsettings.local.json",
+                    true) // Optionally load local settings.
+                .Build();
+
+            _activityName = _configuration["ActivityName"];
```

Comment:
There are 3 config values loaded, but 6 in the file. Are some not used?

Reply:
I was going to use the expanded functions added to the scenario but then decided to use a simpler state machine for testing purposes as described in the comments for testing CreateStateMachine.